Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -58,6 +58,9 @@ static char* timeZone;
 static const char* featureList = nullptr;
 static gboolean enableITP;
 static gboolean printVersion;
+static const char* windowSize;
+static guint windowWidth = 1280; // Default width, when using old API.
+static guint windowHeight = 720; // Default height, when using old API.
```

Comment:
I would initialize these to 0, and then use the default value for old api when command line option is not used (they are still 0).

Reply:
They are also used in the new api when --size is not passed

Reply:
Fixed.